homework/week-0/AllisonBojarski/Cards(HWproblem#2)/src/nyc/c4q/alizinha/Game.java

```diff
@@ -0,0 +1,10 @@
+package nyc.c4q.alizinha;
+import java.util.ArrayList;
+
+/**
+ * Created by c4q-Allison on 5/2/15.
+ */
+abstract public class Game
+{
+    abstract ArrayList getPieces();
```

Comment:
I do believe that this was supposed to be the point which we used a concept of generics to get a random type for the ArrayList such as ArrayList<E>.